### ISSUE TITLE
Flow Features

### ISSUE THREAD

**Issue by srikanthmalla** [MoonBlvd/tad-IROS2019#7]
Hi Brian,
Could you point me to the script where you did ROIPooling of flow features and saved it for different datasets?

Thank you,
Srikanth

**Comment by MoonBlvd** [comment]
@srikanthmalla Yes, if you go to lib/utils/data_prep_utils.py there is ```roi_pooling_opencv``` function, which might be what you are looking for.

**Comment by srikanthmalla** [comment]
1  Did you try ROI pooling of other feature layers instead of output flow image? 
2. Can bilinear interpolation be called as ROI pooling, I think it should be called ROI Align right to avoid confusion? https://everitt257.github.io/blog/2019/02/07/RoI-Explained.html

**Comment by MoonBlvd** [comment]
1. Not as I remember. But I tried pool from intermediate layers of semantic segmentation and it didn't help.
2. I think the bilinear interpolation in ROIAlign is different about what we did. If my understanding is correct, ROIAlign used bilinear to solve the problem that the ROIs doesn't align with the conv layer output size. For example if you have a CH*32*32 output but your ROI vertices in original image, e.g. 1280*1280,  is (500, 600),  which cannot be aligned to any coordinate on the 32*32 feature map. But what we did was to use bilinear interpolation during resizing, which is simpler and not related to alignment issue. In fact we simply rounded our scaled ROI coordinates to align with the flow map we are pooling from, simple but effective.
